### PR TITLE
Mbs 8532 use cloud emulators model

### PR DESCRIPTION
**Pull request by dsvoronin**
https://hub.docker.com/repository/docker/avitotech/android-emulator-22
https://hub.docker.com/repository/docker/avitotech/android-emulator-28

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -184,17 +196,21 @@ subprojects {
...
-                                device = com.avito.instrumentation.reservation.request.Device.Emulator.Emulator22
+                            reservation = TestsBasedDevicesReservationConfiguration().apply {
+                                device = CloudEmulator(
+                                    name = "api22",
+                                    api = 22,
+                                    model = "Android_SDK_built_for_x86",
+                                    image = "avitotech/android-emulator-22:24e6ed0ec4",
+                                    cpuCoresRequest = "1",
+                                    cpuCoresLimit = "1.3"
+                                )
```
Comment (eugene-krivobokov) [inline build.gradle.kts]: How do we pass credentials for the cloud?

**Review by eugene-krivobokov — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (dsvoronin) [reply build.gradle.kts]: dockerhub is available without authentication

**Comment by dsvoronin** [comment]
Should wait till next version release, for backward compatibility (CloudEmulator not available in current version)

File: build.gradle.kts [hunk 2]
```diff
@@ -213,17 +229,21 @@ subprojects {
...
-                                device = com.avito.instrumentation.reservation.request.Device.Emulator.Emulator28
+                            reservation = TestsBasedDevicesReservationConfiguration().apply {
+                                device = CloudEmulator(
+                                    name = "api28",
+                                    api = 28,
+                                    model = "Android_SDK_built_for_x86_64",
+                                    image = "avitotech/android-emulator-28:a9b53d28be",
+                                    cpuCoresRequest = "1",
+                                    cpuCoresLimit = "1.3"
+                                )
```
Comment (eugene-krivobokov) [inline build.gradle.kts]: Duplicated values. Let's reuse them.

**Pull request merged**